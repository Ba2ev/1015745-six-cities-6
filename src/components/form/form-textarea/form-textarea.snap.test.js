import React from "react";
import {render} from '@testing-library/react';
import FormTextArea from './form-textarea';

it(`TextArea should render correctly`, () => {
  const handler = jest.fn();

  const {container} = render(
      <FormTextArea handler={handler}/>
  );

  expect(container).toMatchSnapshot();
});
