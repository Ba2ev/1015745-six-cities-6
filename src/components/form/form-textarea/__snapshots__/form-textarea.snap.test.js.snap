// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea should render correctly 1`] = `
<div>
  <textarea
    class="reviews__textarea form__textarea"
    data-testid="textarea"
    id="review"
    name="review"
    placeholder="Tell how was your stay, what you like and what can be improved"
  />
</div>
`;
